### PR TITLE
Add snapshot tests 

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?
As the title says.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__tests__/MiniCart.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { render } from 'test-utils'
+import { fireEvent } from 'react-testing-library'
+
+import MiniCart from './../index'
+
+describe('<MiniCart /> component', () => {
+  const renderComponent = customProps => {
```
Comment (klzns) [inline react/__tests__/MiniCart.test.js]: **Suggestion**: I prefer tests that are repetitive and easy to change than using functions that "make tests" or keep them DRY.

So in this case I would use the render function directly. It's also more explicit what you are trying to test.

```js
render(<MiniCart type="popup" hideContent={false} />)
```

File: react/__tests__/MiniCart.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  it('should be rendered', () => {
+    expect(renderComponent().asFragment()).toBeDefined()
+  })
+
+  it('should match the snapshot in popup mode', () => {
+    const wrapper = renderComponent()
+    const leftClick = { button: 0 }
+    const { getByText } = wrapper
+    fireEvent.click(getByText('Button Test'), leftClick)
+    expect(wrapper.asFragment()).toMatchSnapshot()
```
Comment (klzns) [inline react/__tests__/MiniCart.test.js]: Some line breaks would be good.

I usually follow the Arrange Act Assert.

In this case would be:
```js
const leftClick = { button: 0 }

const { getByText, asFragment } = render(<MiniCart type="popup" hideContent={false} />)
fireEvent.click(getByText('Button Test'), leftClick)

expect(asFragment()).toMatchSnapshot()
```

File: react/__tests__/MiniCart.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  })
+
+  it('should match the snapshot in sidebar mode', () => {
+    expect(renderComponent({ type: 'sidebar' }).baseElement).toMatchSnapshot()
```
Comment (klzns) [inline react/__tests__/MiniCart.test.js]: Same here:

```js
const { baseElement } = render(<MiniCart type="sidebar" hideContent={false} />)

expect(baseElement).toMatchSnapshot()
```

**Review by klzns — APPROVED** [review]

File: react/__mocks__/vtex.dreamstore-icons.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const IconCart = props => <div> Icon Cart {props.children} </div>
```
Comment (theomoura) [inline react/__mocks__/vtex.dreamstore-icons.js]: could you add a `class` for each one? ie: `className="iconcart-mock"`

or you could mock like [this](https://github.com/vtex-apps/product-kit/blob/master/react/__mocks__/vtex.dreamstore-icons.js):

File: react/__mocks__/vtex.store-components/Animation.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const Animation = props => <div>{props.children}</div>
```
Comment (theomoura) [inline react/__mocks__/vtex.store-components/Animation.js]: could you add a `class` for it? ie: `className="animation-mock"`

File: react/__mocks__/vtex.store-components/ProductPrice.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+const ProductPrice = props => <div>Price</div>
```
Comment (theomoura) [inline react/__mocks__/vtex.store-components/ProductPrice.js]: could you add a `class` for it? ie: `className="price-mock"`

**Review by theomoura — APPROVED** [review]

**Review by theomoura — CHANGES_REQUESTED** [review]
View the comentaries please 😄

**Review by theomoura — APPROVED** [review]

**Pull request merged**